benching/mod.ts

```diff
@@ -90,13 +90,15 @@ export function bench(
 }
 
 /** Runs all registered and non-skipped benchmarks serially. */
-export async function runBenchmarks({
-  only = /[^\s]+/,
-  skip = /^\s*$/
-}: BenchmarkRunOptions): Promise<void> {
+export async function runBenchmarks(
+  options?: BenchmarkRunOptions
+): Promise<void> {
+  // Assigning default options
```

Comment:
I prefer handling all the defaults for function parameters in the signature:
```
export async function runBenchmarks({
  only = /[^\s]/,
  skip = /^\s*$/
}: BenchmarkRunOptions = {}): Promise<void>
```

The default ` = {}` after the argument type is the only part that needs to be added for this fix.
This prevents extra function body code and that explicit `Object.assign`.

What do you think?

Reply:
woaw. Sorry, you're totally right fixing this.